content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -412,6 +412,52 @@ forEach([
 When you use this approach, both the testing framework and CI Visibility can tell your tests apart.
 
 
+## Intelligent Test Runner
+
+<div class="alert alert-warning">Intelligent Test Runner is currently in closed beta.</div>
+
+Read more about Intelligent Test Runner here.
+
+To enable Intelligent Test Runner the following environment variables need to be set:
+
+`DD_CIVISIBILITY_AGENTLESS_ENABLED=true` (Required)
+: Enables or disables Agentless mode.<br/>
+**Default**: `false`
+
+`DD_API_KEY` (Required)
+: The [Datadog API key][1] used to upload the test results.<br/>
```

Comment:
I think the links here may be wrong number 1 is 

`[1]: https://github.com/facebook/jest/tree/main/packages/jest-circus`